src/psyclone/parse/kernel.py

```diff
@@ -557,9 +557,10 @@ def get_procedure(ast, name, modast):
         # Search the the meta-data for a SpecificBinding
         for statement in ast.content:
             if isinstance(statement, fparser1.statements.SpecificBinding):
-                # We support either:
+                # We support:
                 # PROCEDURE, nopass :: code => <proc_name> or
                 # PROCEDURE, nopass :: <proc_name>
+                # or nothing, if there is an explicit interface
```

Comment:
Although this is true, in this branch of the `if` we do only support the two things listed. You could move this whole comment out of the if and replace it with a shorter one.

Reply:
Or, you could just revert this file :-)

Reply:
I removed it.